misc/config_tools/configurator/src/pages/Config.vue

```diff
@@ -198,31 +222,49 @@ export default {
       }
       this.updateCurrentFormData()
     },
-    saveScenario(event) {
+    saveScenario() {
+      let msg = ["Scenario xml saved\n",
+        ".xml settings validated\n",
+        "launch scripts generated\n"];
+      let errmsg = ["Scenario xml save failed\n",
+        ".xml settings validate failed\n",
+        "launch scripts generate failed\n"];
+      let step = 0
       let scenarioXMLData = configurator.convertScenarioToXML(
           {
             // simple deep copy
             "acrn-config": JSON.parse(JSON.stringify(this.scenario))
           }
       );
       configurator.writeFile(this.WorkingFolder + 'scenario.xml', scenarioXMLData)
-          .then(() => configurator.pythonObject.validateScenario(this.board.content, scenarioXMLData))
-          .then((result) => {
-            this.errors = result;
-            if (result.length === 0) {
-              let launchScripts = configurator.pythonObject.generateLaunchScript(this.board.content, scenarioXMLData)
-              for (let filename in launchScripts) {
-                configurator.writeFile(this.WorkingFolder + filename, launchScripts[filename])
-              }
+        .then(() => {
+          step = 1
+          configurator.pythonObject.validateScenario(this.board.content, scenarioXMLData)
+        })
+        .then(() => {
+            step = 2
+            let launchScripts = configurator.pythonObject.generateLaunchScript(this.board.content, scenarioXMLData)
+            for (let filename in launchScripts) {
+              configurator.writeFile(this.WorkingFolder + filename, launchScripts[filename])
             }
-          })
-      event.stopPropagation()
-      event.preventDefault()
+        })
+        .then(() => {
+            alert(`${msg.join('')} \n All files successfully saved to your working folder ${this.WorkingFolder}`)
```

Comment:
It would be good to know the names of the files that were saved vs. just saying "all files".

Reply:
Thanks for the comment. 
The detail board info is in ${msg.join('')}